corehq/apps/export/models/new.py

```diff
@@ -476,6 +503,7 @@ def __insert_system_properties(cls, table, properties, top=True):
 
         for static_column in properties:
             index, existing_column = table.get_column(static_column.item.path, static_column.item.transform)
+            (existing_column or static_column).extra_data(**data)
```

Comment:
nice, in a later commit i do copy the columns. i like that, i think i'll go with the `isinstance` so it's super clear that we're trying to update the row column and not any other column